partials/card.hbs

```diff
@@ -3,23 +3,23 @@ Re-usable card for linking to posts
 --}}
 
 <article class="gh-card {{post_class}} {{#if feature_image}}with-image{{else}}no-image{{/if}}">
-    <a class="gh-card-link" href="{{url}}">
+    <div class="row no-gutters border rounded overflow-hidden flex-md-row align-items-center mb-4 shadow-sm h-md-250 position-relative">
         {{#if feature_image}}
+        <div class="col-md d-block">
             <img class="gh-card-image"
                 srcset="{{img_url feature_image size="s"}} 300w,
-                        {{img_url feature_image size="m"}} 600w,
-                        {{img_url feature_image size="l"}} 1000w,
-                        {{img_url feature_image size="xl"}} 2000w"
+                    {{img_url feature_image size="m"}} 600w,
+                    {{img_url feature_image size="l"}} 1000w,
+                   {{img_url feature_image size="xl"}} 2000w"
                 sizes="(max-width: 1000px) 400px, 700px"
                 src="{{img_url feature_image size="m"}}"
                 alt="{{title}}"
             />
+        </div>
         {{/if}}
-        <div class="gh-card-content">
-            <span class="gh-card-meta"><time datetime="{{date format="YYYY-MM-DD"}}">{{date format="D MMM YYYY"}}</time> <span class="bull">&bull;</span> {{reading_time}}</span>
-            <h2 class="gh-card-title">{{title}}</h2>
-            <p>{{excerpt}}</p>
-            <strong>Read more</strong>
+        <div class="col-md p-4 d-flex flex-column position-static gh-card-content">
+            <p class="h3 mb-0 gh-card-title">{{title}}</p>
+            <p class="card-text mb-auto">{{excerpt characters="150"}} &bull;&nbsp;<a class="text-primary" href={{url}}>Read&nbsp;more</a></p>
```

Comment:
Please remove the bullet point before the "Read more". Also, as `.gh-card-link` is no longer being used, please delete any references to that class from `screen.css`.